source/task-item.js

```diff
@@ -105,7 +105,6 @@ class TaskItem extends HTMLElement {
    */
   connectedCallback() {
     // Set attributes for bootstrap.
-    this.setAttribute('class', 'taskNode d-flex flex-row bd-highlight');
     this.setAttribute('draggable', 'true');
```

Comment:
- A little detail, but generally .attribute is the first option to go to before setAttribute
  - Use .draggable = true instead of setAttribute(‘draggable’, ‘true’)
  - Use .className = “put classes here” instead of setAttribute(‘class’, ‘put classes here’)
   - See here - https://quirksmode.org/dom/core/#attributes

Reply:
Updated files to follow this